spyder/plugins/ipythonconsole/widgets/main_widget.py

```diff
@@ -2616,6 +2619,9 @@ def create_ipyclient_for_server(self, server_id):
             # We save the jupyter_api in the client to perform on it operations
             # related to this plugin.
             jupyter_api=jupyter_api,
+            # We save the files_api in the client to get the remote machine
+            # home directoryabsolute path.
```

Comment:
```suggestion
            # home directory.
```

I think it's not necessary to mention the absolute path here.